src/lib/router/readHdaoFeed.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const _ = require('lodash')
+const conseil = require('conseil')
+
+const { getObjktById, paginateFeed } = require('utils')
+
+module.exports = async function readHdaoFeed(req, res) {
+  const rawFeed = await conseil.hDAOFeed()
+  const sortedFeed = _.orderBy(rawFeed, ['hDAO_balance'], ['desc'])
+  const paginatedFeed = paginateFeed(sortedFeed, 0)
+
+  res.json({
+    result: await Promise.all(
+      paginatedFeed.map(async (objkt) => await _mergeHdao(objkt))
```

Comment:
I've tried a bunch of times to get this to finish locally. It always seems to get hung up on a few of the getObjktById requests.

Reply:
I'd say if any of those fail, let them and ignore them